lib/odata_duty/edms.rb

```diff
@@ -50,6 +50,12 @@ def self.to_oas2(is_collection: false)
       OAS_TYPE
     end
 
+    def self.to_value(object, _context)
+      object&.to_str
+    rescue StandardError => e
+      raise InvalidValue, e.message
+    end
```

Comment:
much more performant than what the base class does:

```ruby
    def self.to_value(*args)
      new(*args).__to_value
    end
```

Will cleanup all these classes in a future PR